src/FriendTechProxy.sol

```diff
@@ -43,7 +43,12 @@ contract FriendTechProxy is Ownable {
 
     // If for some reason ERC20 tokens of value gets transferred into this contract, allow withdrawal
     function emergencyWithdraw(address _token, address _to, uint256 _amount) external onlyOwner {
-        IERC20(_token).transfer(_to, _amount);
+        if (_token == 0x0000000000000000000000000000000000000000) {
+            (bool sent, ) = payable(_to).call{value: _amount}("");
```

Comment:
This can be solved by setting a user's balance in the fallback() function. If the user accidentally sends money to the contract they'll receive a balance representing their share of accidental transfers. Then when you withdraw the _to address's balance is sent. Given the nature and purpose of this contract, I don't think my PR is relevant to the future of this project.